Cargo.toml

```diff
@@ -124,3 +124,8 @@ required-features = ["rt", "stm32f429"]
 [[example]]
 name = "rng-display"
 required-features = ["rt", "stm32f407"]
+
+
+[patch.crates-io]
+#embedded-hal = { git = "https://github.com/ryankurte/embedded-hal.git", branch = "feature/spi-transactions" }
+embedded-hal = { path = "../embedded-hal" }
```

Comment:
Uhm... :D

Reply:
yeeah i keep having to change things to try appease the cursed https://travis-ci.org/github/rust-embedded/embedded-hal/builds/691542832